addons/stock_transport/models/fleet_vehicle_model.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class FleetVehicleModelCategory(models.Model):
+    _inherit = 'fleet.vehicle.model.category'
+
+    max_weight = fields.Float(string="Max Weight (Kg)")
+    max_volume = fields.Float(string="Max Volume (m³)")
+
+    def _compute_display_name(self):
+        for record in self:
```

Comment:
probably need a context to avoid using it in a classic fleet context

Reply:
Hello @amoyaux 

Without the "stock_transfer" module installed, I don't think it'll affect into classic fleet flow, so additional context is not necessary.

Thanks.

Reply:
![image](https://github.com/odoo/odoo/assets/5512395/3cc05171-bc8b-4add-afdc-b97bdd54df31)
That's in the fleet app.

Reply:
On top of that the module is in auto_install. So if you manage it independently, you will end up with it.
I'm not sure the module should be autoinstall

Reply:
But you can keep the name get as it is. Since it's a bit the only way to make the difference between vehicule for inventory or not